tests/test_userstory_39.py

```diff
@@ -56,3 +61,75 @@ def test_melding_overlast():
         'object': verblijfsobject['url'],
     })
     assert 'url' in zaak_object
+
+    '''
+    Upload the files with POST /enkelvoudiginformatieobject (DRC)
+    '''
+
+    byte_content = text_file.read() # text_file comes from pytest fixture
+    base64_bytes = b64encode(byte_content)
+    base64_string = base64_bytes.decode('utf-8')
+
+    text_attachment = drc_client.create('enkelvoudiginformatieobject', {
+        'identificatie': uuid.uuid4().hex,
+        'bronorganisatie': '1',
+        'creatiedatum': zaak['registratiedatum'],
+        'titel': 'detailed summary',
+        'auteur': 'test_auteur',
+        'formaat': 'txt',
+        'taal': 'english',
+        'inhoud': base64_string
+    })
+
+    # Test if the EnkelvoudigInformatieObject stored has the right information
+    assert 'creatiedatum' in text_attachment
+    assert text_attachment['creatiedatum'] == zaak['registratiedatum']
+
+    # Retrieve the EnkelvoudigInformatieObject
+    txt_object_id = text_attachment['url'].rsplit('/')[-1]
+    text_attachment = drc_client.retrieve('enkelvoudiginformatieobject', id=txt_object_id)
+
+    # Test if the attached filed is our initial file
+    assert requests.get(text_attachment['inhoud']).content == byte_content
+
+    byte_content = png_file.getvalue()
+    base64_bytes = b64encode(byte_content)
+    base64_string = base64_bytes.decode('utf-8')
+
+    image_attachment = drc_client.create('enkelvoudiginformatieobject', {
+        'identificatie': uuid.uuid4().hex,
+        'bronorganisatie': '1',
+        'creatiedatum': zaak['registratiedatum'],
+        'titel': 'attachment',
+        'auteur': 'test_auteur',
+        'formaat': 'png',
+        'taal': 'english',
+        'inhoud': base64_string
+    })
+
+    '''
+    Link the files to a 'Zaak' with POST /zaakinformatieobjecten (ZRC)
```

Comment:
this relation needs to be moved to DRC instead of ZRC

Reply:
I deleted the model from zrc (see branch : feature/move-zaak-informatie-object-to-drc), and also from 
test_userstory_39.py test (even if it doesn"t pass because of some other reason).